docs/release.md

```diff
@@ -0,0 +1,77 @@
+A "FIWARE release" contains versioned releases of all Generic Enablers found
+within the [catalogue](https://www.fiware.org/developers/catalogue/). All
+components must work seamlessly with the current context broker and each other.
+
+The following timetable is proposed for a typical FIWARE release cycle, where it
+is assumed that releases will follow a monthly cadence.
+
+## Release Prerequisites
+
+Additional GE development for each release cycle should close by the beginning
+of the month - ideally this should mean that all code to be included in the
+FIWARE release has been:
+
+-   Written
+-   Tested
+-   Documented
+-   Labelled as a [SemVer release](GE_Requirements.md#releases) on **GitHub**
+
+Any proposed [new entries](GE_Checklist.md#new-generic-enablers) to the
+catalogue **MUST** have been presented to the TSC
+
+Any component
+[transitioning to a full member](GE_Checklist.md#transition-to-full-membership)
+of the catalogue **MUST** have completed all "Must" requirements and have been
+presented as such to the TSC
+
+## Release timetable
+
+#### Week 1
+
+The formal commencement of the release process will be announced during the
+**first** TSC meeting of the month
+
+-   A list of release candidates will be collated for the TSC and created in a
+    prerelease candidate list on
+    [GitHub](https://github.com/Fiware/catalogue/releases)
+-   For each Generic Enabler
+    -   The latest tagged **GitHub** release is assumed to part of the current
+        release cycle unless explicitly informed otherwise in writing.
+    -   Where no release is found, the latest tag will be proposed.
+    -   Where no tags are found, the latest hash will be proposed.
```

Comment:
better to require a release tag has been made. we do not want laziness

Reply:
We require a tag for the **release**, but we can make do with a githash if necessary when collating the list of things we want to include - i.e. **prerelease**.

The process of collating the list shouldn't be held up just because someone hasn't finished tagging yet - hence the inclusion of two back-up options.